.github/workflows/staging.yml

```diff
@@ -66,4 +66,5 @@ jobs:
           TF_VAR_service_basic_auth_password: ${{ secrets.SERVICE_BASIC_AUTH_PASSWORD }}
           TF_VAR_alert_email_address: ${{ secrets.ALERT_EMAIL_ADDRESS }}
           TF_VAR_aws_account_number: ${{ secrets.AWS_ACCOUNT_NUMBER }}
+          TF_VAR_enable_alerts: ${{ secrets.ENABLE_ALERTS }}
```

Comment:
Does this need to be passed in as a secret variable?  Is it done this way so that we can configure it through setting it as a secret rather than updating the source code each time?

Reply:
Yes - I guess it's not a secret but I do want it to be configurable without changing the source code..

could it be done like this?:

 workflow_dispatch:
    inputs:
      enableAlerts:
        description: "Enable Alerts? (true/false)"
        required: false
        default: "false"

Reply:
I see, makes sense!  Yeah you could do that.  I think you would need to update the Terraform command below to set that variable.

The [docs are useful](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch), you should be able to extract out the variable from the GitHub context